src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/CodeStyleOptions2.cs

```diff
@@ -84,88 +84,88 @@ private static PerLanguageOption2<CodeStyleOption2<bool>> CreateQualifyAccessOpt
+        defaultValue: CodeStyleOption2.TrueWithSilentEnforcement);
 
     internal static readonly PerLanguageOption2<CodeStyleOption2<bool>> PreferInferredTupleNames = CreatePerLanguageOption(
         CodeStyleOptionGroups.ExpressionLevelPreferences,
         "dotnet_style_prefer_inferred_tuple_names",
-        IdeCodeStyleOptions.CommonDefaults.PreferInferredTupleNames);
+        defaultValue: CodeStyleOption2.TrueWithSuggestionEnforcement);
 
     internal static readonly PerLanguageOption2<CodeStyleOption2<bool>> PreferInferredAnonymousTypeMemberNames = CreatePerLanguageOption(
         CodeStyleOptionGroups.ExpressionLevelPreferences,
         "dotnet_style_prefer_inferred_anonymous_type_member_names",
-        IdeCodeStyleOptions.CommonDefaults.PreferInferredAnonymousTypeMemberNames);
+        defaultValue: CodeStyleOption2.TrueWithSuggestionEnforcement);
 
     internal static readonly PerLanguageOption2<CodeStyleOption2<bool>> PreferIsNullCheckOverReferenceEqualityMethod = CreatePerLanguageOption(
         CodeStyleOptionGroups.ExpressionLevelPreferences,
         "dotnet_style_prefer_is_null_check_over_reference_equality_method",
-        IdeCodeStyleOptions.CommonDefaults.PreferIsNullCheckOverReferenceEqualityMethod);
+        defaultValue: CodeStyleOption2.TrueWithSuggestionEnforcement);
 
     internal static readonly PerLanguageOption2<CodeStyleOption2<bool>> PreferConditionalExpressionOverAssignment = CreatePerLanguageOption(
         CodeStyleOptionGroups.ExpressionLevelPreferences,
         "dotnet_style_prefer_conditional_expression_over_assignment",
-        IdeCodeStyleOptions.CommonDefaults.PreferConditionalExpressionOverAssignment);
+        defaultValue: CodeStyleOption2.TrueWithSilentEnforcement);
 
     internal static readonly PerLanguageOption2<CodeStyleOption2<bool>> PreferConditionalExpressionOverReturn = CreatePerLanguageOption(
         CodeStyleOptionGroups.ExpressionLevelPreferences,
         "dotnet_style_prefer_conditional_expression_over_return",
-        IdeCodeStyleOptions.CommonDefaults.PreferConditionalExpressionOverReturn);
+        defaultValue: CodeStyleOption2.TrueWithSilentEnforcement);
 
     internal static readonly PerLanguageOption2<CodeStyleOption2<bool>> PreferCompoundAssignment = CreatePerLanguageOption(
         CodeStyleOptionGroups.ExpressionLevelPreferences,
         "dotnet_style_prefer_compound_assignment",
-        IdeCodeStyleOptions.CommonDefaults.PreferCompoundAssignment);
+        defaultValue: CodeStyleOption2.TrueWithSuggestionEnforcement);
 
     internal static readonly PerLanguageOption2<CodeStyleOption2<bool>> PreferSimplifiedInterpolation = CreatePerLanguageOption(
         CodeStyleOptionGroups.ExpressionLevelPreferences,
         "dotnet_style_prefer_simplified_interpolation",
-        IdeCodeStyleOptions.CommonDefaults.PreferSimplifiedInterpolation);
+        defaultValue: CodeStyleOption2.TrueWithSuggestionEnforcement);
```

Comment:
i'm going to trust you inlined these all carefully and the original values were preserved.

Reply:
I tried :). Tests actually failed in one case where I made mistake.

Reply:
excellent.